### PR TITLE
Feature/general cleanup

### PR DESCRIPTION
**Pull request by CloudNiner**

### CODE REVIEW

**Comment by CloudNiner** [comment]
Mostly addresses #72 

Further improvements could be time/cost prohibitive.

File: src/nih_wayfinding/app/scripts/views/navigate/navigate-controller.js [hunk 1]
```diff
@@ -156,8 +156,7 @@
                   map.setZoom(13);
                 });
                 Notifications.show({
-                    text: 'Select a the destination you\'d like to be routed to',
-                    timeout: 3000
+                    text: 'Select a the destination you\'d like to be routed to'
```
Comment (flibbertigibbet) [inline src/nih_wayfinding/app/scripts/views/navigate/navigate-controller.js]: While you're in here, looks like there's a typo here ("a the")

**Comment by flibbertigibbet** [comment]
+1

File: src/nih_wayfinding/app/scripts/views/navigate/navigate-controller.js (hunk 1 above)
Comment (CloudNiner) [inline src/nih_wayfinding/app/scripts/views/navigate/navigate-controller.js]: OH MAN! That was the one I originally saw that made me make the changes I did, but then couldn't find again.

**Pull request closed without merging**